### DISCUSSION TITLE
What is the best way to add a table in the admin section

### DISCUSSION THREAD

**Discussion by Schrijvers123** [joomengine/Joomla-Component-Builder#926] [Extention Issues] [answered]
I made a component where i can add trainingen. If i.m edit a training i likt to have an tab that contains a  table with information when an quotation ws enterd. It may not eddited. It's a kind of back log. The infomation in the table comanes from a database table and connectin to an other applicatin (via API). What is the best way to achieve this?

**Comment by Llewellynvdm** [comment, marked as answer]
You could try using the custom tab approach, inside the admin view under the settings tab:

![image](https://user-images.githubusercontent.com/5607939/177342863-486a9aa1-bbf1-4d18-9525-817a453a2e04.png)
